libs/editor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -1431,6 +1431,10 @@ public void onMediaUploadFailed(final String localMediaId, final String errorTyp
         mUploadingMediaProgressMax.remove(localMediaId);
     }
 
+    public void onMediaUploadPaused(final String localMediaId) {
+        mUploadingMediaProgressMax.remove(localMediaId);
+    }
```

Comment:
I like avoiding a need to pass the error type and entangle the paused and error methods, but I worry about modify Aztec. We likely need to test Aztec to avoid regressions. 

Additionally, I note this method lacks an `@Override` annotation that surround methods possess. We should likely follow the existing code pattern. 

Lastly, how confident are we that we only need to remove the `localMediaId`? Do we need the guard clause or other logic from the sibling failed method?

Reply:
Adding `onMediaUploadPaused` to Aztec was done quickly to get around some build errors from Android Studio. I also am not keen on modifying Aztec. I'll do some further testing to see if this is actually needed.

Reply:
My understanding is that it is necessary, as both Aztec and Gutenberg share a common interface to which they must adhere. For Aztec, the latest implementation of this merely passes the invocation onto the existing failed method. So, the regression risk should be fairly low. It is likely still worth testing the Aztec editor via editing classic posts in the app. 

https://github.com/wordpress-mobile/WordPress-Android/blob/b97dbeccb556b565249cfe6d2ab229ba9479d33a/libs/editor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java#L1434-L1438